src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -127,47 +136,65 @@ private bool HasCallerMemberNameAttribute
                                                   && !HasCallerFilePathAttribute
                                                   && HasCallerMemberNameAttribute;
 
-        private ConstantValue DefaultSyntaxValue
+        private ConstantValue DefaultSyntaxValue => LazyDefaultSyntaxValue(out var unused, needsBoundExpression: false);
+
+        private ConstantValue LazyDefaultSyntaxValue(out BoundExpression boundExpressionOpt, bool needsBoundExpression)
```

Comment:
>private ConstantValue LazyDefaultSyntaxValue(out BoundExpression boundExpressionOpt, bool needsBoundExpression) [](start = 8, length = 111)

This is looks significantly more complicated than it should be. Simply extracting relevant portion of MakeDefaultExpression into a helper method and calling it from ExplicitDefaultExpression should work just fine. The only necessary change for DefaultSyntaxValue.get is to call VariableUsePass.Visit.